### ISSUE TITLE
[Feature request] Show spelling mistakes

### ISSUE THREAD

**Issue by talha131** [akahuku/wasavi#7]
wasavi for Firefox does not underline misspelled words. I tried using `set spell` but to no avail. Is it possible then on issuing `set spell` command, the misspelled words are underlined. I don't mean the full functionality of spell family of commands like showing spelling suggestions.

If the misspelled words are only underlined like the browser does in the text area when wasavi is not running, is good enough.

**Comment by agzam** [comment]
Yeah would be totally awesome to have '[s', ']s', and basic 'z' motions. Maybe possible somehow to piggyback on Chrome's own spellchecker? Not that we desperately need it, but it would be cool to have it someday.

**Comment by akahuku** [comment]
Sorry for late reply.
All modern browsers has embedded spell checker, but I think Chrome has not publish spell checking API or something to javascript.
Therefore, that functionality on wasavi probably has to be made from scratch (of course, If a good spell-checking library written in javascript exists, I'd use it).
